hphp/test/slow/include/fast-stat-cache.php.expect

```diff
@@ -0,0 +1 @@
+OldOldNew
```

Comment:
#1447 is because the file system mtime resolution is seconds in the old ext2 file system, so that a file may be changed but it's mtime if not changed. I had added a PR to fix this: #7926.